Formula/git-branchless.rb

```diff
@@ -25,6 +25,10 @@ def install
     system "cargo", "install", *std_cargo_args(path: "git-branchless")
   end
 
+  def post_install
+    system "git", "branchless", "install-man-pages", man1
+  end
```

Comment:
Can we do this in `install`?

Reply:
we can't, git actually complains that it cannot find `branchless` command.

Reply:
Have you tried doing
```ruby
ENV.prepend_path "PATH", bin
```
?

Reply:
that is a good point 👍